public/pages/LogTypes/utils/constants.ts

```diff
@@ -23,3 +23,21 @@ export const defaultLogType: LogTypeBase = {
   tags: null,
   category: '',
 };
+
+export const logTypeLabels = {
+  cloudtrail: 'Cloudtrail',
+  dns: 'DNS',
+  vpcflow: 'VPC Flow',
+  ad_ldap: 'Ad/ldap',
```

Comment:
`AD/LDAP` ?

Reply:
On the `Log types` table page, the description for that type is `Ad/ldap logs`, so I went with that capitalization pattern.

Reply:
got it

Reply:
I added screenshots to the PR overview, and here's a screenshot with the descriptions for reference.
![Screenshot 2023-10-12 at 1 24 35 PM](https://github.com/opensearch-project/security-analytics-dashboards-plugin/assets/79280347/7ca1ad80-60a2-4cfd-ab94-1c776bf1fb1d)

Reply:
Adjusted these labels based on conversation with Amardeep.